tasks/arrayToObject.js

```diff
@@ -7,8 +7,8 @@ const newJson = {};
 
```

Comment:
I think it makes more sense to map over cards before anything else and set all the id's to uppercase. This way you only need to call `toUpperCase` once. Bear in mind `map` returns a new array so you have to do:

```
cards = cards.map(card => {
  return Object.assign({}, card, { ... });
});
```